pulsar-client-cpp/lib/MultiTopicsConsumerImpl.cc

```diff
@@ -160,7 +183,7 @@ void MultiTopicsConsumerImpl::subscribeTopicPartitions(const Result result,
                  (int)(conf_.getMaxTotalReceiverQueueSizeAcrossPartitions() / partitions)));
 
     Lock lock(mutex_);
-    topicsPartitions_.insert(std::make_pair(topicName->toString(), partitions));
+    topicsPartitions_.emplace(topicName->toString(), partitions);
```

Comment:
Actually I think the existing `insert` (or the `emplace`) invocation is wrong.
- `insert` or `emplace`: insert, i.e. if the key already exists, the value won't be updated.
- `operator[]`: insert or update

See https://en.cppreference.com/w/cpp/container/map/emplace and https://en.cppreference.com/w/cpp/container/map/operator_at.